assets/schema_input.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://raw.githubusercontent.com/nf-core/metapep/master/assets/schema_input.json",
+    "title": "nf-core/metapep pipeline - params.input schema",
+    "description": "Schema for the file provided with params.input",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "properties": {
+            "condition": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "errorMessage": "Condition name must be provided and cannot contain spaces"
+            },
+            "type": {
+                "type": "string",
+                "pattern": "^(bins)|(assembly)|(taxa)$",
+                "errorMessage": "Microbiome type must be provided and needs to be either 'bins', 'assembly' or 'taxa'"
+            },
+            "microbiome_path": {
+                "type": "string",
+                "format": "file-path",
+                "exists": true,
+                "pattern": "^\\S+$",
+                "errorMessage": "Microbiome path needs to be provided and is not allowed to contain whitespaces."
+            },
+            "alleles": {
```

Comment:
Maybe take advantage of the nf-validation plugin here to check the HLA allele nomenclature: Example see [here](https://github.com/nf-core/epitopeprediction/blob/e173620ac2a10fae8d0a47f597aa723684456deb/assets/schema_input.json#L16)